Orange/widgets/cpewidget.py

```diff
@@ -32,7 +36,11 @@ def sync_properties(self):
         # to the widget
```

Comment:
It turns out that that needs to be done whenever there's a getattr(self,  paramname) (not getattr(self.node, ...) though!) and that happens 3x in the below code in various places and circumstances. It would be less error-prone to make this a method like retrieve_value_from_widget(self, name) which calls the getattr and does the None conversion - then you can use it in all 3 places and no longer need scattered & easy-to-miss checks if content is in NoneUIValues any longer.

The reason why I'm picky about this is that this class is the central binding site between the cpe and vpe params, and it's already complicated enough... need to keep it as simple and straightforward as possible (otherwise whoever needs to fix the next bug ends up wasting hours poring over this file)